imports/client/components/PuzzlePage.tsx

```diff
@@ -250,29 +253,21 @@ class ViewCountDisplay extends React.Component<ViewCountDisplayProps> {
   };
 
   render() {
-    const text = `(${this.props.count} viewing)`;
+    const text = `See ${this.props.count} ${this.props.count === 1 ? 'hunter' : 'hunters'}`;
```

Comment:
I find this text to be a bit more confusing relative to what was there before - "hunters" doesn't clearly scan to me as "people currently looking at this puzzle". "See {count} viewers" maybe?

Reply:
My thinking was to make it more consistent with the "hunter" language used elsewhere, but you're right. I'll change it to "viewer(s)".